packages/calcite-components/src/tests/utils/cssTokenValues.ts

```diff
@@ -15,10 +15,11 @@ export function getTokenValue(token: string): string {
     color$: "rgb(0, 191, 255)",
     hover$: "rgb(255, 105, 180)",
     pressed$: "rgb(44, 44, 44)",
+    selected$: "rgb(156, 89, 209)",
     shadow$:
       "rgb(255, 255, 255) 0px 0px 0px 4px, rgb(255, 105, 180) 0px 0px 0px 5px inset, rgb(0, 191, 255) 0px 0px 0px 9px",
-    "z-index$": "42",
-    "(size|space)$": "42px",
+    "(z-index)$": "42",
+    "(columns|gap|height|offset|radius|size|space|width)": "42px",
```

Comment:
nitpick: for `offset` and `size`, we need to also support `size-x`, `size-y` and `offset-x`, `offset-y` so maybe we could update that here?

Reply:
those are covered by this use-case